hlxsites/merative2
# feat(thought-leadership): Merge feature branch into Main

## Issue

Fixes - N/A

## Description
**Changed**

- Now that we're all wrapped up with adding the Thought Leadership filtering/landing page. It's time to merge these changes into the `main` branch for the launch of this feature.
- This PR merges our feature branch `feat/thought-leadership` into `main`

## Design Specs

- Figma Link - https://www.figma.com/file/dFJwrp6VkUMSirF2KMLRMx/Thought-leadership-library?type=design&node-id=787-25468&mode=design&t=pRNHGuEjJwkvu9Sy-0

## Test URLs

![Screenshot 2023-07-31 at 12 12 42 PM](https://github.com/hlxsites/merative2/assets/1815714/299bff02-5640-48dd-9b9a-ddada801bc81)

- Before (Changes from `feat/thought-leadership`): https://main--merative2--hlxsites.hlx.page/thought-leadership

![Screenshot 2023-07-31 at 12 12 49 PM](https://github.com/hlxsites/merative2/assets/1815714/8e8e0b13-c4cf-4d56-99c8-daf590c7ada2)

- After (Changes from this PR): https://feat-thought-leadership--merative2--hlxsites.hlx.page/thought-leadership
  
## Testing Instruction

- Visit this page (https://feat-thought-leadership--merative2--hlxsites.hlx.page/thought-leadership)
- Ensure that we have everything ready (eg. Content, metadata, et4c) for the launch/publication
